src/common/logo.c

```diff
@@ -249,6 +269,37 @@ static void loadGarudaLogo(FFlogo* logo, bool doColor)
     sprintf(logo->chars[19], FASTFETCH_TEXT_MODIFIER_BOLT"%s                                              "FASTFETCH_TEXT_MODIFIER_RESET, color);
 }
 
+static void loadLogoFromFile(FFlogo* logo, bool doColor, char* pathToFile)
+{
+    strcpy(logo->name, "amogus");
+
+    const char* color = doColor ? "\033[91m" : "";
+    strcpy(logo->color, "\033[91m");
```

Comment:
I don't think we should set the same color for every logo loaded from file. Use the `--color` option to set it.

Reply:
I'm having trouble getting color to work even when using the examples in ```fastfetch --help color```. None of them seem to work, it just prints the color encoding sequence as it is, even on the information lines (OS, host, kernel, etc)

Reply:
This actually was a bug, fixed in the latest commit.